docs/cluster_ns_management.md

```diff
@@ -0,0 +1,132 @@
+## Cluster & Namespace Management
+
+We add [clusters declaratively](https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#clusters) in ArgoCD. 
+
+The ansible playbook will auto generate the cluster spec that is defined within
+the [prod-vars.yaml](https://github.com/AICoE/aicoe-cd/blob/master/vars/prod-vars.yaml).
+
+
+## Deploying to a namespace
+In order for ArgoCD to be able to deploy to your namespace you must allow ArgoCD access to your namespace. This can be done by doing the following: 
+
+1. Give the `argocd-manager` service account (SA) a role with access to the namespace. 
+2. Add the namespace to the cluster spec in ArgoCD 
+
+
+### 1. Add role to SA 
+ArgoCD uses a SA named `argocd-manager` to deploy resources to another cluster/namespace. These SAs need access to the resources it will be deploying, this is done via roles and rolebindings. 
+
+In your namespace, you will need to deploy a rolebinding like the one below: 
+
+```yaml
+apiVersion: authorization.openshift.io/v1
+kind: RoleBinding
+metadata:
+  name: argocd-manager-rolebinding
+  namespace: <application_namespace>
+roleRef:
+  name: <role>
+subjects:
+  - kind: ServiceAccount
+    name: argocd-manager
+    namespace: <sa_namespace>
+```
+
+Fill out `application_namespace`, `role`, and `sa_namespace`. 
+
+> **`application_namespace`**: This is your project namespace. 
+> 
+> **`sa_namespace`**: On PSI OCP4 this will be `aicoe-argocd` and on Data-Hub clusters it will be `argocd-manager`. 
```

Comment:
Should we add a note saying something like 
"where possible make sure that the `argocd-manager` sa is in the `argocd-manager` namespace

Reply:
This section of the document is meant to be general purpose, it would be the responsibility of the SRE folks to create the SA or be part of the process, since we will need to get the token and encrypt it using sre creds.